satellite/payments/stripepayments/stripe.go

```diff
@@ -208,46 +239,62 @@ func (s *service) GetPaymentMethod(ctx context.Context, id []byte) (_ *payments.
 func (s *service) CreateProjectInvoice(ctx context.Context, params payments.CreateProjectInvoiceParams) (_ *payments.Invoice, err error) {
 	defer mon.Task()(&ctx)(&err)
 
-	customerID := string(params.CustomerID)
+	payment, err := s.db.ProjectPayments().GetByProjectID(ctx, params.ProjectID)
+	if err != nil {
+		return nil, stripeErr.Wrap(err)
+	}
+
+	payer, err := s.db.UserPayments().Get(ctx, payment.PayerID)
+	if err != nil {
+		return nil, stripeErr.Wrap(err)
+	}
+
+	customerID := string(payer.CustomerID)
 
 	// create line items
-	_, err = s.client.InvoiceItems.New(&stripe.InvoiceItemParams{
-		Customer:    stripe.String(customerID),
-		Description: stripe.String("Storage"),
-		Quantity:    stripe.Int64(int64(params.Storage)),
-		UnitAmount:  stripe.Int64(100),
-		Currency:    stripe.String(string(stripe.CurrencyUSD)),
-	})
+	_, err = s.client.InvoiceItems.New(
+		&stripe.InvoiceItemParams{
+			Customer:    stripe.String(customerID),
+			Description: stripe.String("Storage"),
+			Quantity:    stripe.Int64(int64(params.Storage)),
+			UnitAmount:  stripe.Int64(100),
+			Currency:    stripe.String(string(stripe.CurrencyUSD)),
+		},
+	)
 	if err != nil {
 		return nil, stripeErr.Wrap(err)
 	}
 
-	_, err = s.client.InvoiceItems.New(&stripe.InvoiceItemParams{
-		Customer:    stripe.String(customerID),
-		Description: stripe.String("Egress"),
-		Quantity:    stripe.Int64(int64(params.Egress)),
-		UnitAmount:  stripe.Int64(100),
-		Currency:    stripe.String(string(stripe.CurrencyUSD)),
-	})
+	_, err = s.client.InvoiceItems.New(
+		&stripe.InvoiceItemParams{
+			Customer:    stripe.String(customerID),
+			Description: stripe.String("Egress"),
+			Quantity:    stripe.Int64(int64(params.Egress)),
+			UnitAmount:  stripe.Int64(100),
+			Currency:    stripe.String(string(stripe.CurrencyUSD)),
+		},
+	)
 	if err != nil {
 		return nil, stripeErr.Wrap(err)
 	}
 
-	_, err = s.client.InvoiceItems.New(&stripe.InvoiceItemParams{
-		Customer:    stripe.String(customerID),
-		Description: stripe.String("ObjectsCount"),
-		Quantity:    stripe.Int64(int64(params.ObjectCount)),
-		UnitAmount:  stripe.Int64(100),
-		Currency:    stripe.String(string(stripe.CurrencyUSD)),
-	})
+	_, err = s.client.InvoiceItems.New(
+		&stripe.InvoiceItemParams{
+			Customer:    stripe.String(customerID),
+			Description: stripe.String("ObjectsCount"),
```

Comment:
is it a valid description?

Reply:
Description stands for name on invoice pdf

Reply:
`Object Count` sounds better to me